src/rslice.rs

```diff
@@ -184,6 +201,23 @@ fn test_rustls_str() {
     }
 }
 
+#[cfg(test)]
+mod tests {
+    use crate::rslice::*;
+    use core::fmt::Write;
+    use std::convert::TryInto;
+
+    #[test]
+    fn test_rustls_str_debug() {
+        let s = "abcd";
+        let rs: rustls_str = s.try_into().unwrap();
+        let mut buf = String::new();
+        write!(buf, "{:?}", rs).unwrap();
+        let want = r#"rustls_str { data: "abcd", len: 4 }"#;
+        assert_eq!(buf, want)
```

Comment:
I would combine these into one line:

```suggestion
        assert_eq!(format!("{:?}", rs), r#"rustls_str { data: "abcd", len: 4 }"#)
```

Reply:
Ah, much better thanks